src/test/java/com/plaid/client/integration/AssetReportGetTest.java

```diff
@@ -0,0 +1,81 @@
+package com.plaid.client.integration;
+
+import com.plaid.client.PlaidClient;
+import com.plaid.client.request.AssetReportGetRequest;
+import com.plaid.client.request.common.Product;
+import com.plaid.client.response.AssetReportCreateResponse;
+import com.plaid.client.response.AssetReportGetResponse;
+import org.junit.Test;
+import retrofit2.Response;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class AssetReportGetTest extends AbstractItemIntegrationTest {
+
+  @Override
+  protected List<Product> setupItemProducts() {
+    return Arrays.asList(Product.ASSETS);
+  }
+
+  @Override
+  protected String setupItemInstitutionId() {
+    return TARTAN_BANK_INSTITUTION_ID;
+  }
+
+  @Test
+  public void testAssetReportGetSuccess() throws Exception {
+    // Create asset report to get a asset report token
+    List<String> accessTokens = Arrays.asList(getItemCreateResponse().getAccessToken());
+    Response<AssetReportCreateResponse> createResponse = AssetReportCreateTest.createAssetReport(client(), accessTokens);
+    String assetReportToken = createResponse.body().getAssetReportToken();
+
+    // Wait till asset report is ready
+    Response<AssetReportGetResponse> response = waitTillReady(client(), assetReportToken);
+
+    // Validate the responses
+    AssetReportGetResponse respBody = response.body();
+    assertSuccessResponse(response);
```

Comment:
These tests are great! One thing I worry about is having such hard constraints in the cases below. If we update sandbox data to include, say, 5 accounts instead of 4, your tests will fail. Can you remove the test cases that are not dependent on user-provided data? For example, it's fine to assert that the user-provided data is the same in the response as it was in the request. But it's probably not a good idea to assume that the number of accounts will always be the same.